.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description of Changes
+
+Please provide a clear and concise description of the changes made in this PR.
```

Comment:
question: should we add some encouragement to add images and videos if the PR touches UI?

Reply:
that's a great suggestion!